octoprint_mrbeam/iobeam/iobeam_handler.py

```diff
@@ -1122,21 +1056,15 @@ def _handle_error_message(self, message):
             raise Exception("ioBeam requested to reconnect. Now doing so...")
         return 1
 
-    def _handle_response(self, message):
+    def _handle_response(self, response):
```

Comment:
separation of concern: The response handler no longer needs to know the whole context of the "response" submap